MetaMask/metamask-extension
# Update Snaps icon in settings search and fix missing icon

## Explanation
This PR is hotfix for one missing icon in settings search and replacement of the Snaps icon.

The issue with missing icon was because of the wrong usage of the property name for `design-system` icons that are not part of FA.
Snaps icon was left unchanged in the settings search and is replaced now with the one already displayed in the settings navigation.

## Screenshots/Screencaps
### Before
![Screenshot 2023-04-25 at 17 39 33](https://user-images.githubusercontent.com/13301024/234337483-21cef100-c2a6-41a8-b063-930433c905b8.png)
![Screenshot 2023-04-25 at 17 52 11](https://user-images.githubusercontent.com/13301024/234337492-11a84383-64cc-48de-b448-b1c0dc34a77a.png)
![Screenshot 2023-04-25 at 17 59 26](https://user-images.githubusercontent.com/13301024/234337494-f5fc12a2-3767-4033-9212-78b0d97a9972.png)

### After
![Screenshot 2023-04-25 at 18 06 07](https://user-images.githubusercontent.com/13301024/234337554-b019b61a-8a9b-4f7f-85b7-ddd572a9a3a7.png)
![Screenshot 2023-04-25 at 18 06 24](https://user-images.githubusercontent.com/13301024/234337558-cd5c9be4-31e8-41d1-ad37-c3494b761722.png)


## Manual Testing Steps
1. Go to settings
2. Search for `Contact`
3. Contact (book) icon should be displayed correctly
4. Search for `Snaps`
5. New Snaps icon should be displayed correctly (box one like in the left-tab-navigation part)

## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
